.github/ISSUE_TEMPLATE.md

```diff
@@ -4,7 +4,11 @@
 
 ### What actually happened?
 
-### What versions of Pillow and Python are you using?
+### What versions of OS, Python and Pillow are you using?
```

Comment:
I might be nitpicking, but the wording of this is clunky to me. An alternate suggestion - 'What are the versions of your OS, Python and Pillow?'

Reply:
Yes, that's an improvement.

Which do you prefer, that or

> What are your OS, Python and Pillow versions?

Reply:
Either sounds fine.

Reply:
Thanks, updated with the shorter one.